backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/AddVmCommand.java

```diff
@@ -1423,6 +1423,7 @@ protected CreateSnapshotFromTemplateParameters buildDiskCreationParameters(DiskI
         tempVar.setParentParameters(getParameters());
         tempVar.setQuotaId(diskInfoDestinationMap.get(image.getId()).getQuotaId());
         tempVar.setDiskProfileId(diskInfoDestinationMap.get(image.getId()).getDiskProfileId());
+        tempVar.setBackup(diskInfoDestinationMap.get(image.getId()).getBackup());
```

Comment:
`diskInfoDestinationMap.get(image.getId())` now appears 5 times, maybe worth refactoring to a variable?

Reply:
Maybe, on a following patch :)

Reply:
how about imageFromDiskInfoDestinationMap?
and does it really worth it :)

Reply:
you can create an issue and label it as a starter issue like we do in vdsm
https://github.com/oVirt/vdsm/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22

Reply:
does it need a new issue? :)

Reply:
That was just a recommendation, you have my "+1".
IMHO no need to add a new issue, either fix it here, or forget about it. Opening a new issue will take longer than to do it :):)
Regarding the name, `imageFromDiskInfoDestinationMap` is too long, a simple `disk` or `destinationDisk` or `diskInfo` will do.

Reply:
this is not about doing something quickly, it's about creating issues to help onboard new contributors

Reply:
This is so simple, that it will not require from new contributor to compile the code, run & test anything, just work with IDE and git :)
Anyway, this comment of mine can be resolved. I just noticed this low priority issue with code duplication and suggested to eliminate it.